### PR TITLE
sdrangel: 6.4.0 -> 6.8.0

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

Also introduce the `librtlsdr` fork of `rtl-sdr`.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/radio/sdrangel/default.nix [hunk 1]
```diff
@@ -61,11 +85,11 @@ mkDerivation rec {
...
-        SDRangel is an Open Source Qt5 / OpenGL 3.0+ SDR and signal analyzer frontend to various hardware.
+      SDRangel is an Open Source Qt5 / OpenGL 3.0+ SDR and signal analyzer frontend to various hardware.
     '';
     homepage = "https://github.com/f4exb/sdrangel";
     license = licenses.gpl3Plus;
-    platforms = platforms.linux;
     maintainers = with maintainers; [ alkeryn ];
+    platforms = platforms.linux;
   };
 }
```
Comment (SuperSandro2000) [inline pkgs/applications/radio/sdrangel/default.nix:95]: Do we have a formatter for meta? I can never remember in which order things should go.

File: pkgs/development/libraries/librtlsdr/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}:
+
+stdenv.mkDerivation rec {
+  pname = "librtlsdr";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "librtlsdr";
+    repo = "librtlsdr";
+    rev = "v" + version;
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/librtlsdr/default.nix:16]: I would prefer if we would stick to "v${version}" which is used more often and would make treewide changes around it easier.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 118886` at c0bdeb5e run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages skipped due to time constraints:</summary>
  <ul>
    <li>pothos</li>
    <li>sdrangel</li>
  </ul>
</details>
<details>
  <summary>25 packages built successfully:</summary>
  <ul>
    <li>cubicsdr</li>
    <li>dabtools</li>
    <li>dump1090</li>
    <li>gnss-sdr</li>
    <li>gnuradio3_7Packages.ais</li>
    <li>gnuradio3_7Packages.gsm</li>
    <li>gnuradio3_7Packages.osmosdr</li>
    <li>gnuradio3_8Packages.osmosdr</li>
    <li>gqrx</li>
    <li>haskellPackages.rtlsdr</li>
    <li>kalibrate-rtl</li>
    <li>librtlsdr</li>
    <li>nrsc5</li>
    <li>python38Packages.pyrtlsdr</li>
    <li>python38Packages.soapysdr-with-plugins</li>
    <li>python39Packages.pyrtlsdr</li>
    <li>python39Packages.soapysdr-with-plugins</li>
    <li>qradiolink</li>
    <li>rtl-ais</li>
    <li>rtl-sdr</li>
    <li>rtl_433</li>
    <li>soapyrtlsdr</li>
    <li>soapysdr-with-plugins</li>
    <li>svxlink</li>
    <li>urh</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 118886` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
28 packages built:
</summary>
<ul>
<li>cubicsdr</li>
<li>dabtools</li>
<li>dump1090</li>
<li>gnss-sdr</li>
<li>gnuradio3_7Packages.ais</li>
<li>gnuradio3_7Packages.gsm</li>
<li>gnuradio3_7Packages.osmosdr</li>
<li>gnuradio3_8Packages.osmosdr</li>
<li>gqrx</li>
<li>haskellPackages.rtlsdr</li>
<li>kalibrate-rtl</li>
<li>librtlsdr</li>
<li>nrsc5</li>
<li>pothos</li>
<li>python38Packages.pyrtlsdr</li>
<li>python38Packages.soapysdr-with-plugins</li>
<li>python39Packages.pyrtlsdr</li>
<li>python39Packages.soapysdr-with-plugins</li>
<li>qradiolink</li>
<li>rtl-ais</li>
<li>rtl-sdr</li>
<li>rtl_433</li>
<li>sdrangel</li>
<li>soapyrtlsdr</li>
<li>soapysdr-with-plugins</li>
<li>svxlink</li>
<li>urh</li>
<li>welle-io</li>
</ul>
</details>

**Pull request merged**